AccessoryManager.cs

```diff
@@ -49,6 +49,18 @@ public int LoadAccessories(string animName, bool saveToCache = false)
                     slot = Db.Get().AccessorySlots.HatHair;
                     cachable = false;
                 }
+                else if (id.StartsWith("headshape_"))
+                {
+                    slot = Db.Get().AccessorySlots.HeadShape;
+                }
+                else if (id.StartsWith("mouth_"))
+                {
+                    slot = Db.Get().AccessorySlots.Mouth;
+                }
+                else if (id.StartsWith("eyes_"))
+                {
+                    slot = Db.Get().AccessorySlots.Eyes;
+                }

```

Comment:
There's starting to be a lot of `else if` here, perhaps putting prefixes and the corresponding slot in a `List<KeyValuePair>` could make the code more readable as the number of accessories the mod can override continues to increase?